p2p/connection.go

```diff
@@ -173,19 +176,21 @@ func (c *MConnection) OnStart() error {
 
 // OnStop implements BaseService
 func (c *MConnection) OnStop() {
-	c.BaseService.OnStop()
```

Comment:
this line can be safely removed

Reply:
@melekes could we perhaps do that in another PR? I'd like to keep this one focused on just OnStop being invoked once :)